### PR TITLE
Query formatting and linting

### PR DESCRIPTION
**Pull request by jenningsanderson**
## Pull Request

DuckDB v1.1.0 now supports reading/writing GeoParquet metadata — this PR updates the query examples.

This is an attempt at cleaning up the example queries and while we should be doing better _testing_ of the queries, we can at least run sqlfluff to lint the queries...

### Docs Preview:

[Docs preview for this PR.](https://dfhx9f55j8eg5.cloudfront.net/how-to/pr/100)

View all staged runs:
https://github.com/OvertureMaps/docs/actions/workflows/publish-pr-to-staging.yml

### CODE REVIEW

**Review by danabauer — APPROVED** [review]
thanks for adding this

File: src/queries/duckdb/seattle_buildings_for_map.sql [hunk 1]
```diff
@@ -1,16 +1,15 @@
...
-FROM read_parquet('azure://release/__OVERTURE_RELEASE/theme=buildings/type=building/*', filename=true, hive_partitioning=1)
-WHERE bbox.xmin > -122.68 AND bbox.xmax < -121.98
-AND bbox.ymin > 47.36 AND bbox.ymax < 47.79
-) TO 'seattle_buildings.geojsonseq'
-WITH (FORMAT GDAL, DRIVER 'GeoJSONSeq', SRS 'EPSG:4326');
+    SELECT
+        names.primary as name,
+        height,
+        level,
+        ST_GeomFromWKB(geometry) as geometry
```
Comment (jeffdefacto) [inline src/queries/duckdb/seattle_buildings_for_map.sql:11]: All caps for consistency with other changes?

```suggestion
        ST_GEOMFROMWKB(geometry) as geometry
```

File: src/queries/duckdb/seattle_land_for_map.sql [hunk 2]
```diff
@@ -1,15 +1,16 @@
...
-WHERE
-    bbox.xmin > -122.68 AND bbox.xmax < -121.98
-    AND bbox.ymin > 47.36 AND bbox.ymax < 47.79
-)
-TO 'seattle_land.geojsonseq'
+    SELECT
+        subtype,
+        class,
+        names.primary AS name,
+        ST_GeomFromWKB(geometry) as geometry
```
Comment (jeffdefacto) [inline src/queries/duckdb/seattle_land_for_map.sql:9]:
```suggestion
        ST_GEOMFROMWKB(geometry) as geometry
```

File: src/queries/athena/changelog_buildings_join.sql [hunk 3]
```diff
@@ -1,10 +1,13 @@
...
-FROM v2024_06_13_beta_0 AS release
-        INNER JOIN changelog_v2024_06_13_beta_0 AS changelog ON release.id = changelog.id
+SELECT
+    release.id,
+    release.names."primary" AS primary_name,
+    release.sources [ 1 ].dataset AS primary_source,
+    change_type,
+    ST_GEOMFROMBINARY(geometry) AS geometry
+FROM v2024_08_20_0 AS release
+    INNER JOIN changelog_v2024_07_22_0 AS changelog ON release.id = changelog.id
```
Comment (jeffdefacto) [inline src/queries/athena/changelog_buildings_join.sql:8]: This is pointing to an out of date changelog and not dynamically pointing at the releases. I'll add a small commit to this PR to fix this.

**Review by jeffdefacto — APPROVED** [review]

File: src/queries/athena/changelog_buildings_join.sql (hunk 3 above)
Comment (danabauer) [reply src/queries/athena/changelog_buildings_join.sql]: hold up. I pointed to an older release specifically for the example in that doc. I have some local edits to that doc to push tonight or tomorrow.

File: src/queries/duckdb/seattle_land_for_map.sql (hunk 2 above)
Comment (jenningsanderson) [reply src/queries/duckdb/seattle_land_for_map.sql]: Thanks @jeffdefacto — even better, with DuckDB v1.1.0, we can just drop these functions :)

**Pull request merged**